components/org.wso2.carbon.identity.api.user.fido2/src/main/java/org/wso2/carbon/identity/application/authenticator/fido2/endpoint/impl/StartRegistrationApiServiceImpl.java

```diff
@@ -30,6 +30,7 @@
 import org.wso2.carbon.identity.application.authenticator.fido2.endpoint.common.FIDO2Constants;
 import org.wso2.carbon.identity.application.authenticator.fido2.endpoint.common.Util;
 import org.wso2.carbon.identity.application.authenticator.fido2.exception.FIDO2AuthenticatorClientException;
+import org.wso2.carbon.identity.application.authenticator.fido2.exception.FIDO2AuthenticatorServerException;
```

Comment:
Do we need this import now?

Reply:
Required as we catch it and throw a client exception from this class